apps/react-18-tests-v8/src/App.tsx

```diff
@@ -1,8 +1,10 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26749&commitId=9d299e9fbc6f53634ad3d39c586e999b461f9ba5&env=prod)

✅ There was 1 screenshots added, 0 screenshots removed, 1044 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- center, image larger.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>